chpl/chpl-api/src/main/java/gov/healthit/chpl/filter/EnvironmentHeaderFilter.java

```diff
@@ -0,0 +1,36 @@
+package gov.healthit.chpl.filter;
+
+import java.io.IOException;
+
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.env.Environment;
+import org.springframework.stereotype.Component;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+@Component
+public class EnvironmentHeaderFilter extends OncePerRequestFilter {
+
+    private Environment env;
+
+    @Autowired
+    public EnvironmentHeaderFilter(Environment env) {
+        this.env = env;
+    }
+
+    @Override
+    protected void doFilterInternal(HttpServletRequest request, HttpServletResponse response, FilterChain filterChain) throws ServletException, IOException {
+        String serverEnvironment = env.getProperty("server.environment") != null ? env.getProperty("server.environment") : "";
```

Comment:
Do we need an update to one of the property files to put this key into it? Maybe the template file? I honestly don't remember where / how/ if we store property files in git anymore